Trajectory:
    dt: 0.05
    type: "Elliptic" #select between "Elliptic", "Lissajous", "Waypoints"
    gamma: [5.0, 5.0, 5.0] # Gamma parameters for x,y and yaw

Elliptic:
    cosFreq: 0.05
    sinFreq: 0.05
    cosAmp: 2
    sinAmp: 3
    line_of_sight: True # Will make the desired yaw to be tangent to (x,y) if enabled

Lissajous:
    # See here how a and b affect the trajectory: https://fr.wikipedia.org/wiki/Courbe_de_Lissajous
    a: 1.0
    b: 2.0
    cosFreq: 0.05
    sinFreq: 0.05
    cosAmp: 1.75
    sinAmp: 1.0
    line_of_sight: True # Will make the desired yaw to be tangent to (x,y) if enabled

Waypoints:
    waypoint_time: [10, 10, 10, 10]
    waypoint_x: [1, 1, 1, 2]
    waypoint_y: [0, 1, 1, 1]
    waypoint_yaw: [0, 0, 60, 90] # in degrees
    line_of_sight: False # Will make the desired yaw to be tangent to (x,y) if enabled
